Blish HUD/GameServices/GameIntegration/Gw2InstanceIntegration.cs

```diff
@@ -219,7 +219,7 @@ public sealed class Gw2InstanceIntegration : ServiceModule<GameIntegrationServic
                 }
 
                 try {
-                    this.CommandLine = newProcess.GetCommandLine();
+                    this.CommandLine = newProcess.ReadArguments();
                 } catch (Win32Exception e) {
```

Comment:
I am curious about what exception gets thrown here if `ReadArguments()` fails.  I'll review the lib to double-check.

Reply:
Yeah, reviewing the source it looks like there are a couple of possible outcomes.  All are ultimately probably equally unrecoverable as far as I can tell.  Could we change the catch to just catch on `Exception` and slightly revise the logger line message?